[16.0][ADD] pos_bypass_global_discount (#15)
trisdoan/pos: pos_bypass_global_discount/readme/DESCRIPTION.rst

> Field `bypass_global_discount` was introduced in [base_global_discount](https://github.com/OCA/server-backend/tree/16.0/base_global_discount). It's used to allow products to be not taken into account when calculating the global discounts.

Reviewer: ```suggestion
Field `bypass_global_discount` was introduced in [base_global_discount](https://github.com/OCA/server-backend/tree/16.0/base_global_discount). It's used to exclude some products from global discounts calculations.
```

> This module makes sure POS also honors the field when configured on products, to be used with native Discount feature at Order level.

Reviewer: ```suggestion
This module is meant to be installed as a companion to `sale_global_discount`. It makes sure the POS will also honors the `bypass_global_discount` field when user inputs a Discount at the Order level.
```
